springboot/src/main/java/com/codecool/terraformingmarsforum/service/UserPostService.java

```
@@ -16,4 +18,9 @@ public class UserPostService {
     public List<UserPost> getAllUserPosts() {
         return userPostRepository.findAll();
     }
+
+    public void addCommentToPostByPostId(Long postId, Comment comment) {
+        Optional<UserPost> post = userPostRepository.findById(postId);
+        post.ifPresent(leaguePost -> leaguePost.getComments().add(comment));
```

And we need to throw exception if not found